src/docker-osx-dev

```diff
@@ -440,11 +444,24 @@ function init_docker_host {
 
 #
 # Installs rsync on the Boot2Docker VM, unless it's already installed.
+# If the main repository is down, rsync will be installed from a mirror
+# If the mirror is down, download rsync binary and place in Boot2Docker VM
 #
 function install_rsync_on_docker_host {
   log_info "Installing rsync in the Docker Host image"
 
-  $DOCKER_HOST_SSH_COMMAND "if ! type rsync > /dev/null 2>&1; then tce-load -wi rsync; fi"
+  local readonly MIRROR="http://distro.ibiblio.org/tinycorelinux/6.x/x86_64/tcz"
+  local readonly DIR="/tmp/tce/optional/"
+  local readonly ACL="acl.tcz"
```

Comment:
These are the dependencies of rsync when installed via tce-load.
http://distro.ibiblio.org/tinycorelinux/6.x/x86/tcz/rsync.tcz.dep